bower.json

```diff
@@ -15,7 +15,7 @@
     "polymer",
     "cosmoz"
   ],
-  "license": "Apache 2.0",
```

Comment:
Since we have a generic license, I think it should be stated here rather than an URL.
But the old text is wrong, according to https://github.com/bower/spec/blob/master/json.md#license and it should be `Apache-2.0`